src/components/StandaloneDirectorChange/CODDate.vue

```diff
@@ -119,8 +119,8 @@ export default class CodDate extends Mixins(DateMixin) {
    * Local helper to change date from YYYY-MM-DD to YYYY/MM/DD.
    * @returns The formatted date.
    */
-  private formatDate (date: string): string {
-    if (!this.isValidDate(date, '-')) return ''
+  private formatDate (date: string, validate = true): string {
+    if (validate && !this.isValidDate(date, '-')) return ''
```

Comment:
Due to [bug 10519](https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/10159), the min and max dates would fail validation.
With this change, we will show the invalid dates (instead of blank). This should make it obvious when there is a date issue.